docs/integrations/destinations/README.md

```diff
@@ -0,0 +1,14 @@
+import ConnectorRegistry from '@site/src/components/ConnectorRegistry';
+
+# Destinations
+
+A destination is a data warehouse, data lake, database, or an analytics tool where you want to load your ingested data.
+
+Learn about our [Connector Support Levels](/integrations/connector-support-levels.md) to understand what to expect from a connector.
+
+
+## Destinations
+
+<ConnectorRegistry type="destination"/>
+
+_[View the connector registry in full](https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html)_
```

Comment:
```suggestion
_[View the connector registry in full](/integrations)_
```

I think we'd prefer to point folks to the view here https://docs.airbyte.com/integrations.  The list on connectors.airbyte.com is probably too information dense (and doesn't scroll right).

Reply:
Makes sense!